app/src/main/java/com/vci/vectorcamapp/landing/presentation/LandingViewModel.kt

```diff
@@ -97,4 +100,12 @@ class LandingViewModel @Inject constructor(
             }
         }
     }
+
+    private fun observeIncompleteSessionsCount() {
+        viewModelScope.launch {
+            sessionRepository.observeIncompleteSessions().collect { sessions ->
+                _state.update { it.copy(incompleteSessionsCount = sessions.size) }
+            }
+        }
+    }
 }
```

Comment:
Move this into the loadLandingDetails function. You should probably not collect the flow. Look into how we did it for the incomplete and complete sessions using the combine function.